README.md

```diff
@@ -53,7 +52,9 @@ See [real device configuration documentation](docs/real-device-config.md).
 
 ### Known problems
 
-After many failures on real devices it could transition to a state where connections are no longer being accepted. To possibly remedy this issue reboot the device. Read https://github.com/facebook/WebDriverAgent/issues/507 for more details.
+- After many failures on real devices it could transition to a state where connections are no longer being accepted. To possibly remedy this issue reboot the device. Read https://github.com/facebook/WebDriverAgent/issues/507 for more details.
+- iPhone/iPad real devices show overlay, which has `Automation Running Hold both volume buttons to stop` text, since iOS/iPadOS 15. It is limitation by XCTest framework. It should not affect taking screenshot API in Appium/WebDriverAgent.
+- iPhone/iPad real devices [requires passcode or touch id](https://github.com/appium/appium/issues/15898#issuecomment-927340411) when they start a XCTest session since iOS/iPadOS 15. No passcode/touch id preference can avoid the behavior.
```

Comment:
requires -> require

Reply:
"No passcode/touch id" preference allows to avoid the behaviour above

Reply:
8292934e80109dc65c55dc7da839100be604311d